src/suit/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "scripts": {
-    "build": "../../node_modules/webpack/bin/webpack.js",
+    "build": "NODE_ENV=development ../../node_modules/webpack/bin/webpack.js",
```

Comment:
@loitly : I'm going to leave the `master` branch in this state after the merge, and then make a release branch in which the last thing I do is take out this override and set `BuildType = "Final"` in app.config.

Reply:
Sure.  This should work fine.